packages/lib/src/paragraph/Paragraph.tsx

```diff
@@ -1,20 +1,35 @@
+import styled from "styled-components";
 import useTheme from "../useTheme";
-import BaseTypography from "../utils/BaseTypography";
 
 const DxcParagraph = ({ children }: { children: React.ReactNode }): JSX.Element => {
   const colorsTheme = useTheme();
 
   return (
-    <BaseTypography
-      as="p"
+    <ParagraphContainer
```

Comment:
Implement the component as we usually do Jiale, use the ThemeProvider!

`<ThemeProvider theme={colorsTheme.paragraph}>`